config/ftbquests/quests/chapters/productive_bees.snbt

```diff
@@ -404,8 +410,8 @@
 			}]
 		}
 		{
-			x: -6.0d
-			y: 5.5d
+			x: -8.5d
+			y: 4.5d
 			subtitle: "Lures in a Slimy Bee in a Swamp Biome"
```

Comment:
by the way, isn't the swamp biome blocked in .../config/skyblockbuilder/common-config.json5
Just tested claims to not work in twilight swamp